src/dom.ts

```diff
@@ -290,6 +290,12 @@ export class ElementHandle<T extends Node = Node> extends js.JSHandle<T> {
     }, deadline, options, true);
   }
 
+  async selectAll(): Promise<void> {
+    const errorOrNeedsInput = await this._evaluateInUtility(({ injected, node }) => injected.selectAll(node), {});
+    if (typeof errorOrNeedsInput === 'string')
```

Comment:
"needs input" part is confusing. Why don't we return error or undefined?

Reply:
I'd stay consistent with fill. Renamed.